mesheryctl/internal/cli/root/app/view.go

```diff
@@ -131,22 +131,20 @@ var viewCmd = &cobra.Command{
 				if response.Applications == nil {
 					return errors.New("application name not provide. Please get an app name and try again. Use `mesheryctl app list` to see a list of applications")
 				}
-				if app.Name == application {
```

Comment:
@piyushsingariya  

removing `if app.Name == application`      
Will give all names from regex search

Meaning when you do
app view MyApp
It will show all apps starting with MyApp -> MyAppone , MyApptwo, MyAPP